tests/test_contrib_loader.py

```diff
@@ -85,6 +85,27 @@ class MyLoader(ItemLoader):
         il.replace_value('sku', [valid_fragment], re=sku_re)
         self.assertEqual(il.load_item()['sku'], u'1234')
 
+    def test_self_referencing_loader(self):
+        class MyLoader(ItemLoader):
+            url_out = TakeFirst()
+
+            def img_url_out(self, (rel_url,)):
```

Comment:
This syntax is no longer supported in Python 3.x (see http://legacy.python.org/dev/peps/pep-3113/) - could you please fix it, so that we'll have to do less changes?